tests/test_readme.py

```
@@ -0,0 +1,180 @@
+from pathlib import Path
+
+
+class TestREADMEStructure:
+    """Test README.md file structure and content."""
+    
+    def test_readme_file_exists(self):
+        """Test that README.md file exists in project root."""
+        readme_path = Path(__file__).parent.parent / "README.md"
+        assert readme_path.exists(), "README.md file must exist in project root"
+    
+    def test_readme_has_essential_sections(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_readme_has_essential_sections`

Add return type annotation: `None`

(ANN201)